civis/ml/_model.py

```diff
@@ -631,7 +643,10 @@ def __init__(self, model, dependent_variable,
                  cross_validation_parameters=None, model_name=None,
                  calibration=None, excluded_columns=None, client=None,
                  cpu_requested=None, memory_requested=None,
-                 disk_requested=None, verbose=False):
+                 disk_requested=None, notification_urls=None,
+                 success_email_subject=None, success_email_body=None,
+                 success_email_addresses=None, failure_email_addresses=None,
+                 stall_warning_minutes=None, verbose=False):
```

Comment:
OK, I'm happy making the change to `notifications` here. Should I also make the change to `resources` as well or open a new PR and do it there?